third-party-podspecs/RNReanimated.podspec.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "RNReanimated",
-  "version": "2.4.1-wp-1",
+  "version": "2.4.1-wp-2",
```

Comment:
Looks like we forgot to update the version of Reanimated after merging the drag & drop functionality.